nrf_modem/doc/dectphy.rst

```diff
@@ -0,0 +1,296 @@
+
+For example, with modulation scheme 0 (MSC 0) a transmission with data length of 136 / 8 = 17 bytes would take exactly 2 sub-slots.
+Using the same modulation scheme, a transmission of 3 sub-slots will transmit 264 / 8 = 33 bytes, therefore the length of the data being transmitted must be exactly 33 bytes.
+
+.. table:: Bits per subslot index with given modulation scheme
+
+   +---------------------+------+------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+
+   | MSC / Subslot-index | 0    | 1    | 2     | 3     | 4     | 5     | 6     | 7     | 8     | 9     | 10    | 11    | 12    | 13    | 14    | 15    |
+   +---------------------+------+------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+
+   | MCS 0               | 0    | 136  | 264   | 400   | 536   | 664   | 792   | 920   | 1064  | 1192  | 1320  | 1448  | 1576  | 1704  | 1864  | 1992  |
+   +---------------------+------+------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+
+   | MCS 1               | 32   | 296  | 552   | 824   | 1096  | 1352  | 1608  | 1864  | 2104  | 2360  | 2616  | 2872  | 3128  | 3384  | 3704  | 3960  |
+   +---------------------+------+------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+
+   | MCS 2               | 56   | 456  | 856   | 1256  | 1640  | 2024  | 2360  | 2744  | 3192  | 3576  | 3960  | 4320  | 4768  | 5152  | 5536  | --    |
+   +---------------------+------+------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+
+   | MCS 3               | 88   | 616  | 1128  | 1672  | 2168  | 2680  | 3192  | 3704  | 4256  | 4768  | 5280  | --    | --    | --    | --    | --    |
+   +---------------------+------+------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+
+   | MCS 4               | 144  | 936  | 1736  | 2488  | 3256  | 4024  | 4832  | 5600  | --    | --    | --    | --    | --    | --    | --    | --    |
+   +---------------------+------+------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+
+
+Note that for higher MCS levels, a better Signal-to-Noise Ratio (SNR) is needed to correctly decode the transmissions.
+Depending on transmission power and range, higher MCS levels may not be reached.
+
+It is recommended to use transmission lengths of at maximum 4-6 slots.
+
+Hybrid ARC
+----------
+
+The application can schedule a hybrid ARC response transmission (HARQ feedback) by calling the :c:func:`nrf_modem_dect_phy_tx_harq` function.
+A HARQ response transmission can be scheduled after a reception (successful or unsuccessful) on the physical control channel (PCC).
```

Comment:
Unsuccesfull PCC do not require HARQ feedback (we do not know if the packed was for us in such case) hence if may be more accurate to say that HARQ response may be scheduled after successfull/unsuccessful PDC